src/v0.4/parseExpr_staged.jl

```diff
@@ -305,32 +305,16 @@ end
 @generated addToExpression_reorder(ex, arg) = :(addToExpression(ex, 1.0, arg))
 
 @generated function addToExpression_reorder(ex, args...)
-    if !isleaftype(ex) || mapreduce(t -> !isleaftype(t), |, args)
-        error("Can't process abstract types")
-    end
-    # how many of the multiplicands are variables?
-    n_var = mapreduce(t -> (t == Variable || t == AffExpr), +, args)
-    has_quad = mapreduce(t -> (t == QuadExpr), |, args)
```

Comment:
Can you figure out any reason why it was explicitly checking for quadratic terms? Maybe I had some idea to reorder to make quadratics efficient.